backend/src/main/java/com/carffeine/carffeine/station/controller/review/dto/ReviewResponse.java

```diff
@@ -0,0 +1,16 @@
+package com.carffeine.carffeine.station.controller.review.dto;
+
+import com.carffeine.carffeine.station.domain.review.Review;
+import lombok.Builder;
+
+import java.time.LocalDateTime;
+
+@Builder
+public record ReviewResponse(Long reviewId,
```

Comment:
얘도 개행 부탁드려요
그리고 빌더 어노테이션을 사용하신 이유가 있나요? 사용하는 곳이 없어보여서요
```suggestion
public record ReviewResponse(
          Long reviewId,
```

Reply:
중간에 바꾼걸 적용 못햇네요 빼겟습니다